.pipelines/templates/add-arc-extension.yaml

```diff
@@ -1,11 +1,17 @@
+parameters: 
+  - name: imageTag
+    type: string
+    default: $(CHECKOUT_TAG)
+
 steps:  
   - bash: |
+      echo ${{ parameters.imageTag }}
```

Comment:
Can you add more context to this log message. Something like "Image tag: $parameters.imageTag"